charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses/nutanix-cluster-class.yaml

```diff
@@ -3,7 +3,7 @@ kind: KubeadmConfigTemplate
 metadata:
   labels:
     cluster.x-k8s.io/provider: nutanix
-  name: nutanix-quick-start-kcfg-0
+  name: nutanix-quick-start
```

Comment:
will this get overwritten when we do <pre>
make apis.sync
</pre>
ref https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/templates/cluster-template-clusterclass.yaml#L4

Reply:
Good point. That won't do it, but the related `make examples.sync` will.

I guess I can work around this in our kustomization, but also fix it upstream.